### ISSUE TITLE
Bedrock chunk

### ISSUE THREAD

**Issue by FoxFireX** [ImplementsLegend/VaultFaster#5]
Just heard about this mod and wanted to give it a try. Installed latest beta, 2.0.3.1. Generation was astoundingly better, but did find one issue during the vault I tested in. One chunk didn't appear to generate correctly, and was nothing but a solid hunk of bedrock. Don't see anything interesting in the log files, unfortunately.

![2024-08-15_22 02 49](https://github.com/user-attachments/assets/d2071471-ebc0-494c-a2db-f6a0010c371b)

Would love to see this get posted over in the Discord's fan-creations and fan-made-addtions channels. This is really impressive work, and I would love the dev team themselves to maybe try to improve their own generation based on what you've done here.

**Comment by ImplementsLegend** [comment]
How close was it to start room?

In my own testing I have generated millions of chunks and only 1 was broken like this, it was in room right next to the start room and on one of previous versions so I hoped it was related to some other issue that has since been fixed.

**Comment by FoxFireX** [comment]
It might have been close-ish, but I can't swear for sure. The path I always take is to go forward from the portal room into the first "real" room, then turn left and go until I reach the edge of the vault. I'll then head one more row "forward" (same direction as it took to go from the portal to the first real room) and head back the way I came. Timer shows 3:11 on the clock, so this should be after I "made the turn", but I'm not sure just how long after. If I had to guess, I'd expect it was probably about five or six rooms lateral from the centerline of the vault, but I have no data to verify that.
